tests/testthat/helper-shinytest2.R

```diff
@@ -0,0 +1,176 @@
+library(shinytest2)
+library(glue)
+library(htmltools)
```

Comment:
Could we use `shiny::tags` instead of this library?

Reply:
`htmltools` is not used at all. I will remove it.